testing/kuttl/e2e-other/streaming-standby/03--standby-cluster.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
+metadata:
+  name: standby-cluster
+  namespace: kuttl-test-streaming-cluster
```

Comment:
> question This and the primary cluster are both in the kuttl-test-streaming-cluster namespace. Can we just use the one created by KUTTL for this test?

This may have been leftover from when I had them in separate namespaces. I can look into using the default.

> question If so, can this test move to the main e2e directory?

I was thinking that we would have one set of "standby" tests that include a repo-based standby. In that case we would need to have it in `e2e-other`. Should we just have these two tests, streaming-standby and repo-standby, in different places :question:

Reply:
I move the test and added a quick readme :+1: